README.Mac.md

```diff
@@ -1,294 +1,289 @@
+## 2.0 Install PHP extension: ext-ast ^0.1.5 for phan/phan
+Type into your terminal and hit enter:
+```
+pecl install ast-0.1.5;
+```
 
-### 3.5.1 Including vhosts
-Open the ```/etc/apache2/httpd.conf``` in a text editor and locate the line containing ```Include /private/etc/apache2/extra/httpd-vhosts.conf```
+## 2.1 Install Node.js
+Navigate in your browser to [NodeJS.org](https://nodejs.org/en/) to download and run the installer.
 
-Ensure that the line is uncommented (remove the ```#``` at the beginning of the line)
 
-### 3.5.2 Adding new vhost
-Open the ```/etc/apache2/extra/httpd-vhosts.conf``` in the text editor and add the following:
+## 2.2 Install LORIS
+Paste into your terminal and hit enter:
+```
+cd ~/Development/GitHub/Loris
+make dev
+cd /tools
+./install.sh
 ```
-<VirtualHost *:80>
-	ServerAdmin webmaster@localhost
-	DocumentRoot "/Users/$username/Sites/loris/htdocs/"
-	ServerName localhost.loris
-	ErrorLog "/private/var/log/apache2/loris.com-error_log"
-	CustomLog "/private/var/log/apache2/loris.com-access_log" common
-	
-	<Directory "/Users/$username/Sites/loris/htdocs/">
-		Options Indexes FollowSymLinks
-		AllowOverride All
-		Order allow,deny
-		Allow from all
-	</Directory>
-	
-	php_value include_path .:/usr/share/php:$pathToPear:/Users/$username/Sites/loris/project/libraries:/Users/$username/Sites/loris/php/libraries:/usr/local/lib/php/Smarty
-	# Possible values include: debug, info, notice, warn, error, crit,
-	# alert, emerg.
-	LogLevel warn
-
-	ServerSignature Off
 
-	#SSLEngine Off  # change to On to enable, after updating cert paths below
-	#SSLCertificateFile /etc/apache2/ssl/loris-cert.pem
-	#SSLCertificateKeyFile /etc/apache2/ssl/loris-key.pem
-	#SSLCACertificateFile /etc/apache2/ssl/CA-cacert.pem
-</VirtualHost>
+## 2.2.1 Launch the LORIS installer in your browser
+Open a browser (we officially support Firefox & Chrome) and navigate to:
 
-<VirtualHost *:80>
-	ServerName localhost
-	DocumentRoot /Library/WebServer/Documents/
-</VirtualHost>
 ```
- * <i>$username ⇾ your username for the computer</i>
- * <i>$pathToPear ⇾ path where PEAR installs .php files (see [section 1.3.4](#include_path) part 1)</i>
-
-## 3.6 Update Database
+localhost:8080/installdb.php
+```
 
-Note that the default Loris setup assumes that Loris is running on localhost. If this
-is not the case, you'll have to manually update the URL and Host config variables in the
-ConfigSettings table by running the following SQL commands from a MySQL prompt:
+or
 
-```SQL
-UPDATE Config SET Value='$yourURL' WHERE ConfigID=(SELECT ID FROM ConfigSettings WHERE Name='url');
-UPDATE Config SET Value='$yourHostname' WHERE ConfigID=(SELECT ID FROM ConfigSettings WHERE Name='host');
+```
+localhost:8000/installdb.php
 ```
 
-Make sure that `$yourURL` above contains the "http://" or "https://" and `$yourHostname` does not. If your server is only being accessed from localhost, you can skip this step.
+and depending if using php -S or Apache.
```

Comment:
Order of port number is reverse in regard with tools listed

Reply:
@PapillonMcGill I'm not quite sure what you mean here.

Reply:
Also I don't see any more changes from @cmadjar or others.

Reply:
@maltheism 
```
localhost:8080/installdb.php

or

localhost:8000/installdb.php

and depending if using php -S or Apache.
```
8080 is Apache,   
8000 is php -S

either reverse port number line or reverse `php -S` and `Apache`

Reply:
for the unresolved issues, look under the ` hidden conversation`
![unresolved](https://user-images.githubusercontent.com/34311645/54125699-12df4900-43dc-11e9-86ce-beba5fdeefb8.png)

some issues are not even hidden

Reply:
Marking them as resolved yourself and not resolving is not helping

Reply:
@PapillonMcGill I did look in the hidden conversations, can you please re-comment on them thanks! Also maybe @cmadjar can comment on what's not resolved.

Reply:
@PapillonMcGill The ones I marked resolved were when I added the changes I believe. I'll use the thumbs up next time. Sorry!